src/main/webapp/app/delete-dialog/delete-dialog.component.html

```diff
@@ -0,0 +1,27 @@
+<form name="deleteForm" (ngSubmit)="confirmDelete()" #deleteForm="ngForm">
+    <div class="modal-header">
+        <h4 class="modal-title" jhiTranslate="entity.delete.title">Confirm delete operation</h4>
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true" (click)="clear()">&times;</button>
+    </div>
+    <div class="modal-body">
+        <jhi-alert-error></jhi-alert-error>
+        <p [jhiTranslate]="deleteQuestion" [translateValues]="{ title: entityTitle }">Are you sure you want to delete?</p>
+        <div class="form-group" *ngIf="deleteConfirmationText">
+            <label class="text-danger" [jhiTranslate]="deleteConfirmationText">Please type in the name to confirm</label>
+            <input type="text" class="form-control" name="confirmExerciseName" value="" [(ngModel)]="confirmExerciseName" />
+        </div>
+    </div>
+    <div class="modal-footer">
+        <button type="button" class="btn btn-secondary" data-dismiss="modal" (click)="clear()">
+            <fa-icon icon="ban"></fa-icon>&nbsp;<span jhiTranslate="entity.action.cancel">Cancel</span>
+        </button>
+        <button
+            type="submit"
+            class="btn btn-danger"
+            [disabled]="deleteForm.form.invalid || (confirmExerciseName != entityTitle && deleteConfirmationText)"
+            id="delete-quiz-confirmation-button"
+        >
+            <fa-icon icon="times"></fa-icon>&nbsp;<span jhiTranslate="entity.action.delete">Delete</span>
```

Comment:
I never understood what the use of `&nbsp;` is. couldn't this be also a margin?

Reply:
It's a non-breaking space.

Reply:
But then it could also be `flex-row nowrap`, correct? HTML should not contain styling imo.